### ISSUE TITLE
ISOM 2017 - Rounded baseline on cliffs

### ISSUE THREAD

**Issue by ryan-mooore** [OpenOrienteering/mapper#1988]
If I create a new ISOM 2017 map, I would expect the default symbol set to respect the following rule for ISOM from 202:

> Ends of the base line must be rounded if no tags appear.

Symbol 202.1 should therefore have rounded ends as it will never have tags.

From 201:
> Ends of the top line may be rounded or square.

It would be nice to see then variants of this symbol with rounded or squared ends as they are both valid symbols for the mapper to choose from.

**Comment by krticka** [comment]
Yes, you are right, for 202 Cliff caps should be rounded for better distinction between footpath and cliffs. There is an easy fix for this, see the picture.
![Clipboard01](https://user-images.githubusercontent.com/15969434/132810310-9d1b9231-4978-44a0-aea0-2d244f129daa.jpg)

For the 202.1 showing minimum size 0.65 mm (OM) it is more tricky as the rounded ends are adding extra length (0.125 mm on every side) so the line length must me redefined as shown in the picture.
![Clipboard02](https://user-images.githubusercontent.com/15969434/132811220-cf681240-ee94-47b0-8804-08d0791db997.jpg)

When I have more time, I'll try to fix it.

**Comment by Symbian9** [comment]
> Yes, you are right, for `202 Cliff` caps should be `rounded` for better distinction between footpath and cliffs. There is an easy fix for this, see the picture.

@krtichka, should be there also *"Offset at start"* and *"Offset at end"*? What does spec says about that?

**Comment by krticka** [comment]
Specification does not says about offsets, it is developers task to define map symbols. But your note is correct at least for line variant, automatic legibility checks will then work correctly.

**Comment by Symbian9** [comment]
> But your note is correct at least for line variant

Yeah, I mean exactly `202 Cliff` line symbol.

@krticka, would you commit those changes to related symbol set source?

- https://github.com/OpenOrienteering/mapper/tree/master/symbol%20sets/src
